spring-pulsar-spring-boot-autoconfigure/src/main/java/org/springframework/pulsar/autoconfigure/PulsarProperties.java

```diff
@@ -59,6 +60,8 @@ public class PulsarProperties {
 
 	private final Producer producer = new Producer();
 
+	private final Administration administration = new Administration();
```

Comment:
I am in favor of `spring.pulsar.admin` vs. `spring.pulsar.administration`. The reason is that it is much easier to type when doing props and it is just as informative (ie we don't lose anything by chopping it down). We could achieve that by naming this var as such. Wdyt?

Reply:
+1 for `spring.pulsar.admin`.